ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>"It's designed for a latency-sensitive service..."</p><p> </p><p>Something is missing from the the last sentence: "For example, if you have a model to predict if a user is going to watch" Do you mean, "For example, for a movie service, you might want to quickly shows movies that the current user would most likely watch by using online predictions." or something like that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>